### ISSUE TITLE
Cannot override CSS settings as described in docs

### ISSUE THREAD

**Issue by adammhaile** [mmistakes/jekyll-theme-basically-basic#65]
## Description

Trying to override the default theme with a couple of tweaks, primarily a few colors and the font. I followed the docs and created https://github.com/adammhaile/UselessCalculations/blob/master/assets/stylesheets/main.scss
You can see I added a few color tweaks to that. But nothing shows up when the site is generated.
I admit, I'm new to Jekyll, so I'm probably missing something. But everything I'm seeing in [your docs](https://github.com/mmistakes/jekyll-theme-basically-basic#customizing-sass-scss) and the jekyll docs seem to just say to copy the desired base file to the same path, and add the tweaks you want. I also tried with main.scss empty other than my tweaks as seems to be implied in your docs. But same issue.

## Environment

- Basically Basic version: 1.4.0
- Ruby gem or remote theme version: gem 2.5.2.1
- Jekyll version: 3.8.2
- Git repository URL: https://github.com/adammhaile/UselessCalculations/
- Operating system: Ubuntu 18

---

## Expected behavior

For the colors and fonts to be updated by updating /assets/stylesheets/main.scss

## Steps to reproduce the behavior

- Create /assets/stylesheets/main.scss
- Add SASS tweaks
- rebuild

**Comment by mmistakes** [comment]
Move the Sass variables before the `@import` lines and it should work.
